test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe VotesController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
```

Comment:
There are some interesting test cases around voting that I wish you had had time to address.